terraform/modules/tf_stream_alert_globals/classifier_queue/sqs.tf

```diff
@@ -1,6 +1,10 @@
 // SQS Queue: Send logs from the Classifier to the SQS queue
+locals {
+  classifier_prefix = "${var.prefix}_"
```

Comment:
I would either put the ternary operator here, or omit the use of this local altogether.